src/test/java/com/google/sps/data/backgroundTaskManagerInterfaceTest.java

```diff
@@ -0,0 +1,39 @@
+package com.google.sps;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import org.junit.After;
+import org.junit.Before;
+import com.google.sps.data.Instance;
+import com.google.sps.data.InstanceInterface;
+import java.util.Optional;
+
+/** Class that test the methods in the BackgroundTaskManagerInterface class. */
+@RunWith(JUnit4.class)
+public class InstanceTest {
+  private final LocalServiceTestHelper helper =
+  new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+
+  @Before
+  public void setUp() {
+   helper.setUp();
+  }
+
+  @After
+  public void tearDown() {
+   helper.tearDown();
+  }
+
+  @Test
+  public void testGetter() {
+    // Creates test data. 
+    BackgroundTaskManagerInterface backgroundManager = new backgroundManager();
+    Assert.assertEquals(backgroundManager.updateInstances(), "Unimplemented");
+    Assert.assertEquals(backgroundManager.deleteInactiveAttendees(), "Unimplemented");
+    Assert.assertEquals(backgroundManager.replaceFaultyInstances(), "Unimplemented");
```

Comment:
Did you run this test? I dont think this actually compiles. None of these functions return anything so assertEquals should show an error, plus backgroundManager() doesnt exist. Make sure you run the test before putting out a PR.

Look at the question in this stackoverflow thread. *Ignore* the answer. Look at the code in teh question. That's how you should test that something throws and exception. 
https://stackoverflow.com/questions/156503/how-do-you-assert-that-a-certain-exception-is-thrown-in-junit-4-tests